src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+
+    public Circle(double radius, Color color) {
+        this.type = getClass().getSimpleName();
+        this.radius = radius;
+        this.color = color;
```

Comment:
Its better to use super class constructor
```suggestion
        super(color);
```